tests/compliance_operator_v2_test.go

```diff
@@ -45,3 +46,40 @@ func TestComplianceV2ProfileGet(t *testing.T) {
 	}
 	assert.Greater(t, len(profile.Rules), 0, "failed to verify ocp4-cis profile contains any rules")
 }
+
+func TestComplianceV2ScheduleRescan(t *testing.T) {
+	conn := centralgrpc.GRPCConnectionToCentral(t)
+	client := v2.NewComplianceScanConfigurationServiceClient(conn)
+
+	sc := v2.ComplianceScanConfiguration{
+		Id:       uuid.NewV4().String(),
+		ScanName: "CIS scan schedule",
+		ScanConfig: &v2.BaseComplianceScanConfigurationSettings{
+			OneTimeScan: false,
+			Profiles:    []string{"ocp4-cis", "ocp4-cis-node"},
+			ScanSchedule: &v2.Schedule{
+				IntervalType: 1,
+				Hour:         0,
+				Minute:       0,
+				Interval: &v2.Schedule_DaysOfWeek_{
+					DaysOfWeek: &v2.Schedule_DaysOfWeek{
+						Days: []int32{1, 2, 3, 4, 5, 6, 7},
+					},
+				},
+			},
+			Description: "Scan schedule for CIS profiles to run daily.",
+		},
+		// TODO: Figure out what to put here
+		Clusters: []string{},
+	}
+	scanConfig, err := client.CreateComplianceScanConfiguration(context.TODO(), &sc)
```

Comment:
This will need permissions on the cluster so that the sensor service account can actually create the CO resources.